tests/wamr-test-suites/test_wamr.sh

```diff
@@ -453,7 +453,7 @@ function spec_test()
     echo -e "\nFinish spec tests" | tee -a ${REPORT_DIR}/spec_test_report.txt
 }
 
-function wasi_test()
+function wasi_test_OUTDATED()
```

Comment:
at this point it has very different coverage to simply replace.

Reply:
Agree, I'd rather keep this name unchanged, and maybe find another name for the new test suite (`wasi_certification_tests()`?). Once we have the same coverage, we can delete this test suite completely.

Reply:
and for the new `wasi_certification_tests`, should I just skip the coverage generation? I don't think we can use the same (`lcov`) as in the other tests?

Reply:
I'd keep the integration simple for now, and we can add more features in the future if needed

Reply:
It's not an official concept (at least not yet), but we strive to provide an extensive coverage so the testsuite can act as certification tests.